docs/content/docs/updates/v1.0.0.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Fluid Framework 1.0
+aliases:
+  - /updates/1.0/
+summary: |
+  The 1.0 release is the first major release of the Fluid Framework.
+---
+
+## Patch Updates
+
+### Update 1.0.1:
+- Add the missing `ConnectionState` export to `fluid-framework`
+- Fix AzureClient issue where the second user could not connect in `local` mode
+
+## Breaking changes
+
+### connected() on FluidContainer is replaced with connectionState
+
+The `connected()` method has been replaced with the `connectionState` property. Read more
+
+```js
+import { ConnectionState } from "fluid-framework";
+
+// Previous Syntax
+if (container.connected) {
+// New Syntax
+if (container.connectionState === ConnectionState.Connected) {
+  console.log("Container is connected");
+}
+```
+
+### AzureClient connection configuration has distinct local and remote connection strings
+
+The AzureClient connection config has been updated to have distinct types to differentiate between local and remote connections. There is now an `AzureRemoteConnectionConfig` and `AzureLocalConnectionConfig` type that is strongly typed to the requirements of each connection. You will notice that since the `tenantID` is not need in the local case it is not longer required on the config.
```

Comment:
```suggestion
The AzureClient connection config has been updated to have distinct types to differentiate between local and remote connections. There is now an `AzureRemoteConnectionConfig` and `AzureLocalConnectionConfig` type that is strongly typed to the requirements of each connection. You will notice that since the `tenantID` is not needed in the local case, it is no longer required on the config.
```